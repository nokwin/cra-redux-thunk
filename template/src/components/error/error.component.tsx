import React from "react";

export const Error = () => <h1>Oooops! Something went wrong</h1>;
