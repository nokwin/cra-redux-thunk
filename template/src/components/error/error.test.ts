import { Error } from "./error.component";

describe("Error component", () => {
  it("should render error component", () => {
    expect(Error).toMatchSnapshot();
  });
});
