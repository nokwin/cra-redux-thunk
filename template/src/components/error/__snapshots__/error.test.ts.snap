// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error component should render error component 1`] = `[Function]`;
